geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionClearWithExpirationDUnitTest.java

```diff
@@ -281,6 +283,19 @@ private void registerVMKillerAsCacheWriter(List<VM> vmsToBounce) {
     }));
   }
 
```

Comment:
I don't know what the code change here caused the test PartitionedRegionClearWithExpirationDUnitTest to take 72 minutes to finish, while it took 17 minutes(still quite long) in GEODE-7665 branch

Reply:
I have run this on machine many a times, and it finishes in 11 or 12 min. I am guessing it could be an environment issue.